### DISCUSSION TITLE
release 2.4.1

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#537] [Team Posts]
Hi!
Could you please release spring-data-aerospike?

Regarding automated build: We can't use `com.aerospike.spring` group id as it will be published under `com.aerospike` root, so it will require keys for `com.aerospike` group id. @rbotzer What do you think?

**Comment by arrowplum** [comment]
Hi,

We are going to make the release. I think we have to brainstorm on how to do this in a more friendly manner.

Best,

Joe

**Comment by Aloren** [comment]
Hi Joe. Thanks for the response. We can setup release ourselves, but we need to know exactly whether you are ok with us publishing to `com.aerospike.spring` as it will require configuring your secret variable for maven publishing. If you are not comfortable with that, then we should definitely change the group id to smth like `com.playtika`. Or you can setup release on your side and give us only a release button :) So please let me know what approach you prefer more :)

**Comment by Aloren** [comment]
Ronen recently noted that we can use Github Packages, unfortunately it does not suit us as it requires authentication for artifacts download.
